README.md

```diff
@@ -0,0 +1,39 @@
+# graal-jvmci-8
+
+Fork of jdk8u/hotspot with support for JVMCI.
+
+
+## Building JVMCI JDK 8
+
+To create a JVMCI enabled JDK 8 on other platforms (e.g., Windows):
```

Comment:
other?

Reply:
I literally copied the section over...will fix this.

Reply:
Ok, great. This document needs to read correctly in a stand alone context.

Reply:
PTAL

Reply:
?

Reply:
Oh sorry, [PTAL](https://en.wiktionary.org/wiki/PTAL).

Reply:
Aren't abbreviations a wonderful time saver ;-)